e2e-tests/tfm/cypress/e2e/pages/dealsPage.js

```diff
@@ -3,6 +3,7 @@ const dealsPage = {
   searchFormInput: () => cy.get('[data-cy="search-input"]'),
   searchFormSubmitButton: () => cy.get('[data-cy="submit-button"]'),
   dealsTableRows: () => cy.get('[data-cy="deals-table"] tbody tr'),
+  dealIdCell: (rowNumber) => cy.get('[data-cy="deals-table"] tbody tr').eq(rowNumber).children('td').eq(0),
```

Comment:
Should this be in`dealsTable.row` below? Looks like other body cells/columns are in there